docs/resources/uberbank_example1.ini

```diff
@@ -134,46 +139,39 @@ approximant = SEOBNRv2_ROM_DoubleSpin_HI
 iterative-match-df-max = 8.0
 cache-waveforms =
 
+mtotal-max = 100
+mratio-min = 1
+mratio-max = 97.989
+
+aligned-spin =
+bh-spin-min = -${workflow|max-bh-spin}
+bh-spin-max = ${workflow|max-bh-spin}
+ns-spin-min = -${workflow|max-ns-spin}
+ns-spin-max = ${workflow|max-ns-spin}
+ns-bh-boundary-mass = ${workflow|nsbh-boundary}
+
 [sbank-bbh]
 ; These options are sent to all sbank "BBH" jobs
 
 ; Mass/spin parameter options
```

Comment:
That choice makes sense, however if you really want to keep the increase in template count small, I think you have to restrict the overcovered BBH region to m2 > 10 or something like that. Otherwise you also overcover part of the NSBH space, where you have many templates, and then the increase is no longer small. Let's discuss similar ideas on Wednesday's call maybe.